.github/workflows/industrial_ci_action.yml

```
@@ -11,85 +11,38 @@ on:
-        python3-pytest-cov \
-        python3-rosdep \
-        python3-setuptools \
-        python3-vcstool \
-        wget \
-        clang-format-10 && \
-        python3 -m pip install -U \
-        argcomplete \
-        flake8-blind-except \
-        flake8-builtins \
-        flake8-class-newline \
-        flake8-comprehensions \
-        flake8-deprecated \
-        flake8-docstrings \
-        flake8-import-order \
-        flake8-quotes \
-        pytest-repeat \
-        pytest-rerunfailures \
-        pytest && \
-        set +u && source /opt/ros/foxy/setup.bash && \
-        colcon build --cmake-args -DCMAKE_BUILD_TYPE=Release && set -u
-      AFTER_SETUP_UPSTREAM_WORKSPACE: "vcs pull $BASEDIR/upstream_ws/src"
-      AFTER_SETUP_UPSTREAM_WORKSPACE_EMBED: "set +u && source ~/ws_moveit/install/setup.bash && set - u"
-      AFTER_RUN_TARGET_TEST: ${{ matrix.env.CCOV && './.ci.prepare_codecov' || '' }}
-      TARGET_CMAKE_ARGS: >
-        -DCMAKE_BUILD_TYPE=${{ matrix.env.CCOV && 'RelWithDebInfo' || 'Release'}}
-        ${{ matrix.env.CCOV && '-DCMAKE_CXX_FLAGS="--coverage" --no-warn-unused-cli' || '' }}
-      CCACHE_DIR: "${{ github.workspace }}/.ccache"
+      AFTER_SETUP_UPSTREAM_WORKSPACE: vcs pull $BASEDIR/upstream_ws/src
+      AFTER_SETUP_UPSTREAM_WORKSPACE_EMBED: set +u && source ~/ws_moveit/install/setup.bash && set - u
+      TARGET_CMAKE_ARGS: -DCMAKE_BUILD_TYPE=Release
+      CCACHE_DIR: ${{ github.workspace }}/.ccache
       BASEDIR: ${{ github.workspace }}/.work
-      CACHE_PREFIX: "${{ matrix.env.IMAGE }}${{ matrix.env.CCOV && '-ccov' || '' }}"
-      CLANG_TIDY_BASE_REF: "${{ github.base_ref || github.ref }}"
+      CACHE_PREFIX: ${{ matrix.env.IMAGE }}
 
-    name: "${{ matrix.env.IMAGE }}${{ contains(matrix.env.IKFAST_TEST, 'ikfast') && ' + ikfast' || ''}}${{ matrix.env.CCOV && ' + ccov' || ''}}${{ matrix.env.CLANG_TIDY && ' + clang-tidy' || '' }}"
+    name: ${{ matrix.env.IMAGE }}
     runs-on: ubuntu-latest
     steps:
-      - name: "Free up disk space"
-        if: ${{ matrix.env.CCOV }}
-        run: |
-          sudo apt-get -qq purge build-essential ghc*
-          sudo apt-get clean
-          # cleanup docker images not used by us
-          docker system prune -af
-          # shift ccache folder to /mnt on a separate disk
-          sudo mkdir /mnt/ccache
-          mkdir ${{ env.CCACHE_DIR }}
-          sudo mount --bind ${{ env.CCACHE_DIR }} /mnt/ccache
-          # free up a lot of stuff from /usr/local
-          sudo rm -rf /usr/local/*
-          df -h
       - uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
       - name: cache upstream_ws
         uses: pat-s/always-upload-cache@v2.1.3
         with:
           path: ${{ env.BASEDIR }}/upstream_ws
-          key: upstream_ws-${{ env.CACHE_PREFIX }}-${{ hashFiles('moveit2.repos') }}-${{ github.run_id }}
+          key: upstream_ws-${{ env.CACHE_PREFIX }}-${{ hashFiles('moveit2_tutorials.repos.repos') }}-${{ github.run_id }}
```

This seems to be the wrong file name: *.repos.repos
```suggestion
          key: upstream_ws-${{ env.CACHE_PREFIX }}-${{ hashFiles('moveit2_tutorials.repos') }}-${{ github.run_id }}
```